libs/ui/src/lib/accounts-table/accounts-table.component.stories.ts

```diff
@@ -0,0 +1,215 @@
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatSortModule } from '@angular/material/sort';
+import { MatTableModule } from '@angular/material/table';
+import { RouterModule } from '@angular/router';
+import { IonIcon } from '@ionic/angular/standalone';
+import { moduleMetadata } from '@storybook/angular';
+import type { Meta, StoryObj } from '@storybook/angular';
+import { NgxSkeletonLoaderModule } from 'ngx-skeleton-loader';
+
+import { GfEntityLogoComponent } from '../entity-logo';
+import { GfValueComponent } from '../value';
+import { GfAccountsTableComponent } from './accounts-table.component';
+
+const mockAccounts = [
+  {
+    id: '1',
+    name: 'Checking Account',
+    currency: 'USD',
+    balance: 15000,
+    value: 15000,
+    valueInBaseCurrency: 15000,
+    transactionCount: 25,
+    allocationInPercentage: 0.15,
+    isExcluded: false,
+    comment: 'Primary checking account',
+    platform: {
+      name: 'Bank of America',
+      url: 'https://www.bankofamerica.com'
+    },
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    platformId: 'bofa',
+    userId: 'user1'
+  },
+  {
+    id: '2',
+    name: 'Trading Account',
+    currency: 'USD',
+    balance: 5000,
+    value: 125000,
+    valueInBaseCurrency: 125000,
+    transactionCount: 127,
+    allocationInPercentage: 0.65,
+    isExcluded: false,
+    comment: null,
+    platform: {
+      name: 'Interactive Brokers',
+      url: 'https://www.interactivebrokers.com'
+    },
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    platformId: 'ibkr',
+    userId: 'user1'
+  },
+  {
+    id: '3',
+    name: 'Savings Account',
+    currency: 'EUR',
+    balance: 20000,
+    value: 20000,
+    valueInBaseCurrency: 21600,
+    transactionCount: 8,
+    allocationInPercentage: 0.2,
+    isExcluded: false,
+    comment: 'Emergency fund',
+    platform: {
+      name: 'Deutsche Bank',
+      url: 'https://www.deutsche-bank.de'
+    },
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    platformId: 'db',
+    userId: 'user1'
+  },
+  {
+    id: '4',
+    name: 'Excluded Account',
+    currency: 'USD',
+    balance: 1000,
+    value: 1000,
+    valueInBaseCurrency: 1000,
+    transactionCount: 3,
+    allocationInPercentage: 0,
+    isExcluded: true,
+    comment: null,
+    platform: {
+      name: 'Local Credit Union',
+      url: null
+    },
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    platformId: 'lcu',
+    userId: 'user1'
+  }
+];
+
+export default {
+  title: 'Accounts Table',
+  component: GfAccountsTableComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [
+        CommonModule,
+        NgxSkeletonLoaderModule,
+        MatButtonModule,
+        MatMenuModule,
+        MatSortModule,
+        MatTableModule,
+        RouterModule.forRoot([]),
```

Comment:
I get the following error:
```
NG04007: The Router was provided more than once. This can happen if 'forRoot' is used outside of the root injector. Lazy loaded modules should use RouterModule.forChild() instead.
```

Changing it to `RouterModule.forChild([])` works for me.